_config.yml

```diff
@@ -8,12 +8,26 @@ thumbnail_path: "/assets/img/logos-gruntwork/gruntwork-overview.png"
 repository: "gruntwork-io/gruntwork-io.github.io"
 overview_video_embed_url: "//www.youtube.com/embed/dcnDu5Cc0pY"
 
-highlighter: rouge
+highlighter: none
+
+markdown: kramdown
+kramdown:
+   syntax_highlighter_opts:
+      disable : true
```

Comment:
Why disable the markdown syntax highlighter? 

I know you added prism, but if our Markdown / AsciiDoc processors can do the syntax highlighting at build time, producing HTML/CSS that is already highlighted, that will be more performant than loading JS and doing the highlighting at runtime. Or does something about the Markdown / AsciiDoc syntax highlighting not work?

Reply:
I actually thought disabling it would make a difference but it doesn't so I have reverted it